index.js

```diff
@@ -0,0 +1,89 @@
+const bookContainer = document.querySelector('.book-container');
+const holder = document.createElement('div');
+
+const addTitle = document.querySelector('.add-title');
+const addAuthor = document.querySelector('.add-author');
+const button = document.querySelector('.btn');
+
+let books;
+
+const saveBooks = () => {
+  localStorage.setItem('books', JSON.stringify(books));
+};
+
+const savedBooks = JSON.parse(localStorage.getItem('books'));
+if (Array.isArray(savedBooks)) {
+  books = savedBooks;
+} else {
+  books = [];
+}
+
+const createBooks = (title, author) => {
+  const id = `${new Date().getTime()}`;
+  books.push({
+    title,
+    author,
+    id,
+  });
+
+  saveBooks();
+};
+
+const deleteBooks = (idToDelete) => {
+  books = books.filter((book) => {
+    if (book.id === idToDelete) {
+      return false;
+    }
+    return true;
+  });
```

Comment:
[OPTIONAL] Nice work on the book removing functionality :+1:. To make your code shorter here, I suggest you do the following: 
`books = books.filter((book) => book.id !== idToDelete);`